src/main/java/com/example/Restaurant_Management_System_REST_API/configuration/SecurityConfig.java

```diff
@@ -23,13 +23,24 @@ public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity, MvcReq
         return httpSecurity
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(requests -> requests
-                        .requestMatchers(mvc.pattern("")) //TODO: update path with appropriate mapping
-                        .hasRole("ADMIN")
+                        .requestMatchers(mvc.pattern("/api/admin/customer/add")) //TODO: update path with appropriate mapping
+                        .authenticated()
+
+                        //Menu records
                         .requestMatchers(mvc.pattern("/api/menu/record/add"), mvc.pattern("/api/menu/record/update/**"),
                                  mvc.pattern("/api/menu/record/delete/**"))
                         .hasAnyRole("OWNER", "MANAGER")
                         .requestMatchers(mvc.pattern("/api/menu/record/find/**"), mvc.pattern("/api/menu/record/findAll"))
                         .hasAnyRole("OWNER", "MANAGER", "STAFF")
+
+                        //Customer
```

Comment:
This comment made me believe that below are patterns **reachable** for a customer and not the patterns that **include** "customer" in uri path :) My suggestion is to clear out confusion that could make by changing the comment to be more descriptive, but also short.

To code below I have no other comments, it's all looking good.

Reply:
allright! I hope now its more descriptive, thanks for that!